tests/foreman/cli/test_repository.py

```diff
@@ -1884,7 +1884,7 @@ def test_negative_create_unprotected_ostree_repo(self):
 
     @tier2
     @upgrade
-    @skip_if_bug_open('bugzilla', 1658444)
+    @skip_if_bug_open('bugzilla', 1672491)
     def test_positive_synchronize_ostree_repo(self):
         """Synchronize ostree repo
```

Comment:
I'd keep the record of the bugs also in the docstring, like `bz: 1658444, 1672491` it's done a lot around robottelo and I think it's a good practice as it keeps the history of bzs associated with the test even if we no longer skip by it

Reply:
@pondrejk agreed , and updated same.